### PR TITLE
Release v1.1.0 - restrict files published to npm package, add samples for failure response handling

### PR DESCRIPTION
**Pull request by rajkumarpalani**
This PR covers the following

1. Prepare to release https://github.com/paypal/Payouts-NodeJS-SDK/pull/4 which addresses restricting npm package to have necessary files
2. Add samples on handling API failures and how to parse error definition from it
3. Update readme with API failure sample
4. Fix verbiage in payload representation from `5$` to `1$` in README, samples, tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,7 +89,65 @@ let createPayouts  = async function(){
...
+request.requestBody(requestBody);
+
+// Call API with your client and get a response for your call
+let createPayouts  = async function(){
+    try {
+        let response = await client.execute(request);
+        console.log(`Response: ${JSON.stringify(response)}`);
+        // If call returns body in response, you can get the deserialized version from the result attribute of the response.
+        console.log(`Payouts Create Response: ${JSON.stringify(response.result)}`);
+    catch {
```
Comment (aninja8) [inline README.md:140]: catch(e)?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@paypal/payouts-sdk",
-  "version": "1.0.0",
+  "version": "1.1.0",
```
[the remark this answers is not in the document]
Comment (rajkumarpalani) [reply package.json:3]: the change doesn't seem like a patch, so opted for a minor version upgrade

File: README.md (hunk 1 above)
Comment (rajkumarpalani) [reply README.md:140]: fixed

**Review by aninja8 — APPROVED** [review]
LGTM!

**Pull request merged**